app/javascript/scavenger_hunt/scenes/PlayArena/PlayArena.jsx

```diff
@@ -21,12 +20,6 @@ const PlayArena = (props) => {
     return clueId
   }
 
-  if (queryLoading) {
-    return(
-      <h3>Loading your data</h3>
-    )
-  }
-
   if (!queryLoading && !userData.currentUser) {
```

Comment:
Do we have `queryLoading` variable now? should it not be `isLoading`? Which we get from `useProgress` Hook

Reply:
whoops, i didnt see that. thanks il change that.